src/components/MDX/MDXComponents.tsx

```diff
@@ -368,8 +370,28 @@ function YouTubeIframe(props: any) {
   );
 }
 
-function Image(props: any) {
-  return <img className="max-w-[calc(min(700px,100%))]" {...props} />;
+function Image(allProps: any) {
+  const {height, width, style, ...props} = allProps;
+  return (
+    <div
+      style={{
+        position: 'relative',
+        width: !width ? '' : `${width}px`,
+        maxHeight: !height ? '' : `${height}px`,
+        maxWidth: 'calc(min(700px,100%))',
+        ...(!style ? {} : style),
+      }}
+      className={styles.imageContainer}>
```

Comment:
can we rather use tailwind classes? + move the `position: 'relative'` also to classname

Reply:
Yes, using tailwind classes now. I was not able to target the nextImage wrapper using tailwind v3.0, now added a variable NextImageWrapper in the tailwind config.